docs/source/denesting.rst

```diff
@@ -0,0 +1,33 @@
+Denesting of EBU-TT-Live documents
+======================================
+
+DenesterNode should be used when a EBU-TT-3 document has a div that contains
+other divs, or a span contains another span. These elements are not
```

Comment:
_and those nested elements need to be flattened, for example before conversion to EBU-TT-D_